### DISCUSSION TITLE
Weather Stories

### DISCUSSION THREAD

**Discussion by hamdrew** [weather-gov/api#303] [Q&A]
**Is your feature request related to a problem? Please describe.**
I am unable to obtain [weather stories](https://www.weather.gov/mkx/weatherstory) from the NWS API.

**Describe the solution you'd like**
An API that provides active weather stories for a given office, including the title, image, and description.

**Describe alternatives you've considered**
I've considered polling the individual weather story pages and parsing the HTML, but that is prone to break, and I've noticed inconsistencies between weather story pages.

**Additional context**
I want to write a bot that will send me new weather stories when they are published. I find the weather stories to be an enjoyable way to hear about updated forecasts and other weather education.

**Comment by aeharding** [comment]
This is really critical to disseminate weather stories, especially as twitters api is being locked down.

please consider adding support, or if it’s out of scope for this project, please let me know who to contact.

even a super basic RSS feed would work wonders.

**Comment by dwhitemv25** [comment]
In the interim, this is a popular PHP script to fetch and redisplay weather stories from NWS WFO sites and display them on a personal weather template site:

https://www.wxforum.net/index.php?topic=43711.0
